Mathematics/TriangleArea/ggb.py

```diff
@@ -0,0 +1,521 @@
+from IPython.display import display, HTML, Javascript
```

Comment:
I have been unable to figure out how to import ggb while the file ggb.py is not in the same folder as the notebook that is trying to import it. So because I put the triangle and parallelogram notebooks in different folders, they both needed the file in their respective folders. If you know how we can get around that, I would appreciate your help.

Reply:
Maybe I'll try to poke at this a bit myself. I'm pretty new to jupyter notebooks, so I'm not always sure how to fit my square python knowledge into that round hole yet...

Reply:
Like @sean-fitzpatrick mentioned currently, the best way to include the ggb package is to do a `!pip install` at the beginning of the notebook and remove the ggb.py files from the sub directories.